packages/playground/website/playwright/playground-fixtures.ts

```diff
@@ -0,0 +1,50 @@
+import { test as base, Page, expect, FrameLocator } from '@playwright/test';
+
+type WordPressFixtures = {
+	wordpressPage: FrameLocator;
+	page: Page;
+};
+
+export const test = base.extend<WordPressFixtures>({
+	wordpressPage: async ({ page }, use) => {
+		const wpPage = page
+			.frameLocator('#playground-viewport')
+			.frameLocator('#wp');
+		await use(wpPage);
+	},
+	page: async ({ page }, use) => {
```

Comment:
Should we rename this to `websitePage` to make it more clear?

Reply:
topLevelPage maybe? or just `website`?

Reply:
Maybe `webAppPage`?

IMO any of the non-`page` options are fine and would improve clarity.

Reply:
I renamed `page` to `website` and `wordpressPage` to `wordpress`.